stack.yaml

```diff
@@ -1,5 +1,19 @@
 resolver: lts-13.23
 
 extra-deps:
-    - github-0.20
-    - relude-0.5.0
\ No newline at end of file
+  - ansi-terminal-0.9
+  - co-log-0.3.0.0
+  - co-log-core-0.2.0.0
+  - chronos-1.0.5
+  - github-0.20
+  - torsor-0.1 # for chronos
+  - jwt-0.9.0
+  - proto-lens-0.5.0.0
+  - proto-lens-protoc-0.5.0.0
+  - proto-lens-runtime-0.5.0.0
+  - relude-0.5.0
+  - servant-0.16.0.1
+  - servant-server-0.16
+  - servant-proto-lens-0.1.0.3
+  - tomland-1.0.0
+  - typerep-map-0.3.2
```

Comment:
If some library is not specified in `build-depends` then it means that it's used implicitly by some other libraries. But if stackage snapshot doesn't contain those libraries, then we need to add them to `extra-deps`. When this happens, we usually add comment after dependency why it's needed. This makes future cleanup easier. So, for example, all `co-log`-related dependencies in `extra-deps` should look like this (modulo only relevant libraries):

```yaml
# logging
- co-log-0.3.0.0
- ansi-terminal-0.9.1        # for co-log
- chronos-1.0.5              # for co-log
- co-log-core-0.2.0.0        # for co-log
- torsor-0.1                 # for co-log
- typerep-map-0.3.2          # for co-log
```

Reply:
Fixed. Let me know if I did it correctly.